pkg/services/ngalert/remote/remote_primary_forked_alertmanager.go

```diff
@@ -37,19 +37,26 @@ func (fam *RemotePrimaryForkedAlertmanager) GetStatus() apimodels.GettableStatus
 }
 
 func (fam *RemotePrimaryForkedAlertmanager) CreateSilence(ctx context.Context, silence *apimodels.PostableSilence) (string, error) {
-	return "", nil
+	_, err := fam.internal.CreateSilence(ctx, silence)
+	if err != nil {
+		return "", err
+	}
+	return fam.remote.CreateSilence(ctx, silence)
 }
 
 func (fam *RemotePrimaryForkedAlertmanager) DeleteSilence(ctx context.Context, id string) error {
-	return nil
+	if err := fam.remote.DeleteSilence(ctx, id); err != nil {
```

Comment:
I think this function is non-idempotent which is fine so long as it's a conscious decision. I understand we're not thinking about reconciling Alertmanagers on a periodic basis so perhaps it's just something we need to be aware of.